PS C:\Users\NAVYA\Documents\moodify> python intent_classifier.py
2025-02-26 00:12:11.737923: I tensorflow/core/util/port.cc:153] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2025-02-26 00:12:13.232626: I tensorflow/core/util/port.cc:153] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
cuda
train-00000-of-00001.parquet: 100%|████████████████████████████████████████████████████████████████████████| 312k/312k [00:00<00:00, 2.83MB/s]
C:\Users\NAVYA\AppData\Local\Programs\Python\Python311\Lib\site-packages\huggingface_hub\file_download.py:142: UserWarning: `huggingface_hub` cache-system uses symlinks by default to efficiently store duplicated files but your machine does not support them in C:\Users\NAVYA\.cache\huggingface\hub\datasets--clinc_oos. Caching files will still work but in a degraded version that might require more space on your disk. This warning can be disabled by setting the `HF_HUB_DISABLE_SYMLINKS_WARNING` environment variable. For more details, see https://huggingface.co/docs/huggingface_hub/how-to-cache#limitations.
To support symlinks on Windows, you either need to activate Developer Mode or to run Python as an administrator. In order to activate developer mode, see this article: https://docs.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development
  warnings.warn(message)
validation-00000-of-00001.parquet: 100%|█████████████████████████████████████████████████████████████████| 77.8k/77.8k [00:00<00:00, 4.63MB/s]
test-00000-of-00001.parquet: 100%|█████████████████████████████████████████████████████████████████████████| 136k/136k [00:00<00:00, 4.81MB/s]
Generating train split: 100%|████████████████████████████████████████████████████████████████| 15250/15250 [00:00<00:00, 210143.07 examples/s] 
Generating validation split: 100%|█████████████████████████████████████████████████████████████| 3100/3100 [00:00<00:00, 233598.79 examples/s] 
Generating test split: 100%|███████████████████████████████████████████████████████████████████| 5500/5500 [00:00<00:00, 288149.49 examples/s]
Some weights of BertForSequenceClassification were not initialized from the model checkpoint at bert-base-uncased and are newly initialized: ['classifier.bias', 'classifier.weight']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
Epoch 1/3 Training: 100%|███████████████████████████████████████████████████████████████████████████████████| 954/954 [04:26<00:00,  3.57it/s]
Epoch 1/3, Loss: 3449.6031, Train Accuracy: 0.4677
Epoch 2/3 Training: 100%|███████████████████████████████████████████████████████████████████████████████████| 954/954 [04:25<00:00,  3.60it/s] 
Epoch 2/3, Loss: 1115.7661, Train Accuracy: 0.9301
Epoch 3/3 Training: 100%|███████████████████████████████████████████████████████████████████████████████████| 954/954 [04:24<00:00,  3.61it/s] 
Epoch 3/3, Loss: 324.9103, Train Accuracy: 0.9817
Testing: 100%|██████████████████████████████████████████████████████████████████████████████████████████████| 344/344 [00:27<00:00, 12.57it/s] 
Test Accuracy: 0.8800
Precision: 0.8978, Recall: 0.8800, F1-score: 0.8741
PS C:\Users\NAVYA\Documents\moodify> 